script.js

```diff
@@ -1,4 +1,217 @@
+      }
+      break;
+    case LOWEST_COMBI_MODE:
+      if (player1TotalScore > player2TotalScore) {
+        message = `Leaderboard <br> Player 2 is Leading!🎉 <br> Player 2: ${player2TotalScore} <br> Player 1: ${player1TotalScore} <br> Press submit to continue to play`;
+      } else {
+        message = `Leaderboard <br> Player 1 is Leading!🎉 <br> Player 1: ${player1TotalScore} <br> Player 2: ${player2TotalScore} <br> Press submit to continue to play`;
+      }
+      break;
+  }
+  return message;
+};
+
+const switchGameMode = function (userInput) {
+  switch (userInput) {
+    case "high":
+      gameMode = HIGHEST_COMBI_MODE;
+      break;
+    case "low":
+      gameMode = LOWEST_COMBI_MODE;
+      break;
+  }
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  let myOutputValue = "";
+  var input = input.toLowerCase().trim();
+  //input validation for choosing game mode: "High or Low"
+  let correctMode = ["high", "low"];
+  if (correctMode.includes(input)) {
+    switchGameMode(input);
+    console.log("input");
+    console.log(input);
+    console.log("gamemode");
+    console.log(gameMode);
+  } else if (gameMode == HIGHEST_COMBI_MODE) {
+    if (whosTurn == PLAYER_1) {
+      let player1Roll = displayDice();
+      player1DiceCombi.push(combineDice(gameMode));
+      player1TotalScore += player1DiceCombi[0];
+      whosTurn = PLAYER_2;
+      myOutputValue = `Welcome Player 1! <br> ${player1Roll}. <br> <br> Your number combination is ${player1DiceCombi}. <br> <br> ${leaderBoard(
+        gameMode
+      )} <br> It is now Player 2's turn.`;
+    } else if (whosTurn == PLAYER_2) {
+      let player2Roll = displayDice();
+      player2DiceCombi.push(combineDice(gameMode));
+      player2TotalScore += player2DiceCombi[0];
+      console.log("player 2 dice combi");
+      console.log(player2DiceCombi);
+      myOutputValue = `Welcome Player 2! <br> ${player2Roll}. <br> <br> Your number combination is ${player2DiceCombi}. <br> <br> ${leaderBoard(
+        gameMode
+      )} <br> It is now Player 1's turn.`;
+      resetGame();
+    }
+  } else if (gameMode == LOWEST_COMBI_MODE) {
+    if (whosTurn == PLAYER_1) {
+      let player1Roll = displayDice();
+      player1DiceCombi.push(combineDice(gameMode));
+      player1TotalScore += player1DiceCombi[0];
+      whosTurn = PLAYER_2;
+      myOutputValue = `Welcome Player 1! <br> ${player1Roll}. <br> <br> Your number combination is ${player1DiceCombi}. <br> <br> ${leaderBoard(
+        gameMode
+      )} <br> It is now Player 2's turn.`;
+    } else if (whosTurn == PLAYER_2) {
+      let player2Roll = displayDice();
+      player2DiceCombi.push(combineDice(gameMode));
+      player2TotalScore += player2DiceCombi[0];
+      console.log("player 2 dice combi");
+      console.log(player2DiceCombi);
+      myOutputValue = `Welcome Player 2! <br> ${player2Roll}. <br> <br> Your number combination is ${player2DiceCombi}. <br> <br> ${leaderBoard(
+        gameMode
```

Comment:
This 4 if blocks have the same lines of codes, you can write a function to run for them so that you avoid repeating code

Reply:
Thank you for the review and feedback Dom!